app/src/main/AndroidManifest.xml

```diff
@@ -60,7 +60,7 @@
             android:exported="true"
             android:label="@string/app_name"
             android:launchMode="singleTop"
-            android:windowSoftInputMode="stateUnchanged|adjustResize"
+            android:windowSoftInputMode="stateAlwaysHidden|adjustResize"
```

Comment:
This makes it so the keyboard is hidden after the dialog is dismissed.

Reply:
doesnt this make it virtual keyboard always invisible (i.e. not popping up) on mainactivity - so breaks todo/quicknote?

especially, when closing virtual keyboard somehow, and you click again in textfield, does it pop up again?

Reply:
The keyboard will not auto-show on launching DocumentActivity or MainActivity. Returning to DocumentActivity or MainActivity from a dialog, for example will also cause the keyboard to be hidden. A tap will show the keyboard.

Works well on my device. Will test `stateHidden` too and report back

Reply:
Thank you

Reply:
`stateAlwaysHidden` is the behavior I think is best - always hide the keyboard when the editor takes focus. i.e. when opened or when returning from a dialog. User can tap to show keyboard.